Podfile

```diff
@@ -29,6 +29,7 @@ target 'VLC-iOS' do
   target 'VLC for iOSUITests' do
     pod 'SimulatorStatusMagic'
   end
+  target 'VLC for iOSTests'
```

Comment:
I'm not a 100% sure but I think we have three warnings with this addition(for example a the end of `pod install`).